### DISCUSSION TITLE
Manual Installation :  Cannot get HEAD - not a git repository

### DISCUSSION THREAD

**Discussion by seigneurfuo** [nabaztag2018/pynab#476] [General]
Hello.
I don't know if I'm in the right category.
I would need support regarding Pynab.

System: Debian 11 with manual install of pynab 

After following the instructions and once the pynab service is installed, I notice in the upgrade/ updates page and I have the following message on all repositories:

Cannot get HEAD - not a git repository

![image](https://github.com/nabaztag2018/pynab/assets/20669563/89f575d8-1b99-4683-9464-9c11aedb0efb)

I've added the path to the error message by modifying the pynab/nabweb/views.py file. (arround line 532)

The path that displayed correspond to the location of the Git repositories previously specified for installation.

I've also run the command directly from a Python interractive console and it works (with bash too).

Git repositories don't seem to be recognized as such.

Any ideas on how to get Pynab to see its git repositories properly?

Thanks.

**Comment by f-laurens** [comment]
Your problem is probably the one reported here:  https://www.tagtagtag.fr/forum/showthread.php?tid=491, and which is corrected by PR #451.
If this is the case, pending a new Pynab release that includes the fix, a solution is to fix the problem by hand, as detailed here: https://www.tagtagtag.fr/forum/showthread.php?tid=491&pid=2726#pid2726.

**Comment by seigneurfuo** [comment]
I looked in the issues but didn't think to look in pull requests.

Thanks for your quick reply.
